assets/src/migration/README.md

```diff
@@ -0,0 +1,28 @@
+# Migrations
```

Comment:
Just out of interest, why does this live in assets/src/migration and not packages/migration.

Reply:
It will be moved to its own package in a separate ticket.

This PR just lays the foundation.